microsoft-edge/webview2/concepts/end-user-faq.md

```diff
@@ -0,0 +1,70 @@
+---
+title: WebView2 end-user FAQ
+description: Information for end users about WebView2 processes, memory and CPU usage, and automatic installation.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 03/30/2023
+---
+# WebView2 end-user FAQ
+
+This FAQ is for end users, to explain the `msedgewebview2.exe` application on your computer.  If your question isn't listed here, you can click the **Feedback** button in the top-right corner, to enter a question as an [Issue in the edge-developer Docs repo](https://github.com/MicrosoftDocs/edge-developer/issues), and your question might be added to this article.  See also [Contact the WebView2 Team](../contact.md).
```

Comment:
Reeza messaged me a good point - We shouldn't point people to GitHub, this is targeted at people who probably don't use GitHub.